### PR TITLE
RDISCROWD-1670 KPI validation optimize and cleanup

### PR DESCRIPTION
**Pull request by primaryobjects**
- Refactored validation for kpi via WTForms.

### CODE REVIEW

**Comment by fiorda** [comment]
coverage decrease due to deleted lines

File: pybossa/forms/forms.py [hunk 1]
```diff
@@ -97,40 +99,12 @@ class ProjectForm(Form):
     subproduct = SelectField(lazy_gettext('Subproduct'),
                              [validators.Required()], choices=[("", "")], default="")
 
-    kpi_meta = { "min": 0.1, "max": 120, "places": 2, "invalid": 'Number must be between 0.1 and 120.0.' }
-    kpi = DecimalField(lazy_gettext('KPI - Estimate of amount of minutes to complete one task (0.1-120)'),
-        [validators.Required()])
+    kpi = DecimalField(lazy_gettext('KPI - Estimate of amount of minutes to complete one task (0.1-120)'), places=2, rounding=ROUND_UP,
```
Comment (markroseberry) [inline pybossa/forms/forms.py]: I don't think the `rounding` parameter has any effect in our code. I tried submitting the HTML form with different numbers and it always rounded to nearest.

If it did have an effect then why choose `ROUND_UP`? I think we should leave that parameter out unless we understand how it works. It's misleading for the code to say `ROUND_UP` and then not actually do that.

Comment (primaryobjects) [reply pybossa/forms/forms.py]: According to the [documentation](https://wtforms.readthedocs.io/en/stable/fields.html#basic-fields), the DecimalField has a default rounding as mentioned below. I suppose by specifying it explicitly, it's more intentional that leaving it out.

> rounding – How to round the value during quantize, for example decimal.ROUND_UP. If unset, uses the rounding value from the current thread’s context.

Comment (primaryobjects) [reply pybossa/forms/forms.py]: It appears the default behavior of the kpi field is indeed to ROUND_UP. I've removed it to simplify the code. https://github.com/bloomberg/pybossa/pull/274

**Pull request merged**